.gitignore

```diff
@@ -120,3 +120,6 @@ nbproject
 .dir-locals.el
 Build.props
 Directory.Build.props
+
+# optional folder for extension projects
+/ext
```

Comment:
Instead of requiring the `ext` directory to live under the QuantLib root, how about allowing it to live anywhere and passing the name of the directory via a CMake variable such as `QL_ADD_SUBDIRECTORY`? As far as I know, the directory passed to `add_subdirectory` doesn't actually need to be a subdirectory of the project root, and it should be easier to keep the code under its own source control in a separate location.

Reply:
Looks good to me!